apps/spruce/src/pages/task/metadata/index.tsx

```diff
@@ -473,20 +475,30 @@ const DetailsDescription = ({
   status: string;
 }) => {
   const MAX_CHAR = 100;
-
+  const isFailingTask = isFailedTaskStatus(status);
   const baseCopy = description || failingCommand;
-  const fullText =
-    status === TaskStatus.Failed
-      ? `${processFailingCommand(baseCopy, isContainerTask)}`
-      : `Command: ${baseCopy}`;
+  const fullText = isFailingTask
+    ? `${processFailingCommand(baseCopy, isContainerTask)}`
+    : `${baseCopy}`;
+
   const shouldTruncate = fullText.length > MAX_CHAR;
   const truncatedText = fullText.substring(0, MAX_CHAR).concat("...");
 
   return (
     <MetadataItem>
+      {isFailingTask ? (
+        <b
+          css={css`
+            color: ${red.base};
+          `}
+        >
+          Failing Command:{" "}
+        </b>
```

Comment:
nit. it could be nice to keep this as StyledBody or store the css in a variable

It's just a little easier to read because the linter doesn't force it to be on different lines:

<img width="465" alt="Screenshot 2024-08-13 at 2 10 32 PM" src="https://github.com/user-attachments/assets/219af153-78f9-4869-ad47-c7fe0729b1c2">